docs/source/index.rst

```diff
@@ -1,4 +1,4 @@
-Welcome to tus_storagehandler's documentation!
+Welcome to cloud_storage_handler's documentation!
```

Comment:
Should be:

> Welcome to Cloud Storage Handler's documentation!

Presumably this uses the project name from the `.cruft.json` when it is auto-generated. Let's use the human readable name here

Reply:
Assuming we go ahead without waiting to resolve the title/slug issue in the Cookiecutter template asdiscussed [here](https://github.com/elixir-cloud-aai/cloud-storage-handler/pull/17#discussion_r1724903282). What is the better way to deal with these names, in your opinion, @JaeAeich, until ?

- Use the value we want? (`Cloud Storage Handler`)
- Use the value that corresponds to the variable used in the current template? (`cloud_storage_handler`)?

Reply:
let it be as is. We should loose ability to update via cruft so after we get that done in cookiecutter, all this would be fairy easy and automatic, so I don't think we should really worry about this atm.

Reply:
Makes sense